### PR TITLE
[IMP] web: hide seconds in datetime widgets

### PR DESCRIPTION
**Pull request by laro-odoo**
Task-2517675

### CODE REVIEW

File: content/developer/reference/frontend/javascript_reference.rst [hunk 1]
```diff
@@ -707,6 +707,14 @@ Date & Time (`datetime`)
 
             <field name="datetimefield" options="{'rounding': 10}" />
 
+    - `show_seconds`: when set to false, it hides the seconds from the datetime field.
+      The field will still accept datetime values, but the seconds will be hidden in
+      the UI (default: `true`).
+
+        .. code-block:: xml
+
+            <field name="datetimefield" widget="datetime" options="{'show_seconds': false}" />
```
Comment (AntoineVDV) [inline content/developer/reference/frontend/javascript_reference.rst:716]: Is the widget necessary here?

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+ delegate=Polymorphe57

File: content/developer/reference/frontend/javascript_reference.rst (hunk 1 above)
Comment (Polymorphe57) [reply content/developer/reference/frontend/javascript_reference.rst:716]: not always but in a list view yes. So I would keep it here

**Comment by Polymorphe57** [comment]
@robodoo r+

**Pull request closed without merging**